### ISSUE TITLE
Property Tax data on PODP

### ISSUE THREAD

**Issue by prasadpawarr** [opendatapune/Problem-Statements#11]
https://github.com/opendatapune/Problem-Statements/wiki/Property-Tax-data-on-PODP

**Comment by daminisheth** [comment]
working on it

**Comment by prasadpawarr** [comment]
Working on the file "Ward wise Property Tax collection for the year 2016--17"

**Comment by ghost** [comment]
https://drive.google.com/open?id=1KLUjp9BJivRqh8kX5HGxzE4BAPZkuyex

Cleaned csv version of dataset "Ward wise Property Tax collection for the year 2016--17".

**Comment by ghost** [comment]
Group members:
Damini Sheth @daminisheth
Prasad Pawar @prasadpawarr
Krishna Lodha @krishnaglodha

**Comment by answerquest** [comment]
Problem in the submitted CSV : commas in numbers! :fearful:

**Comment by harshadmanglani** [comment]
Team: Playcoders
College: PICT, First Year Engineers
Names and email ids:  
Ayush Das
ayushdas641@gmail.com
Harshad Manglani
harshad.gm@gmail.com
Yash Sonar
yashsonar213@gmail.com 


@answerquest  Here are all the csvs in a github repo.
Link: https://github.com/harshadmanglani/Opendatahackathon

Hereby, all files are cleaned. If there's still anything, please let us know. The geo-spatial map videos are going to be uploaded in the above repo soon.
Website update: Our website is now live, though data uploading and touch up work is still ongoing, will redirect you to the website as soon as its finished.

**Comment by answerquest** [comment]
Good job @harshadmanglani,  team Playcoders! Just check the problem statement again if there were any other excels to clean also regarding property tax?

**Comment by harshadmanglani** [comment]
@answerquest Definitely sir, we're working on csvs, forgot to mention that in the previous comment. Kindly take a look at this geo-spatial map which soon will be uploaded to a legitimate website.

Link to drive: https://drive.google.com/open?id=1GZSx0UmjyXwWkkO08czOht2CyqRXAYYn

**Comment by harshadmanglani** [comment]
@answerquest Yes sir, all files were cleaned and have been uploaded. Here is a video link to our first map. Kindly give your feedback. 
Link: https://drive.google.com/open?id=1hPseNEEKnUxT5dfGrtmS24iOGZD_jDks

Regards,
Team Playcoders.

**Comment by harshadmanglani** [comment]
@answerquest  OUR WEBSITE IS NOW UP AND RUNNING. 
Link: http://playcoders.wixsite.com/punepropertytax
Although still undergoing some tweaks here and there, we are almost done. Will have it ready on/before 28th of February.

Regards,
Team Playcoders.

**Comment by harshadmanglani** [comment]
@answerquest Sir, hereby we, Team Playcoders, give our final submission in the form of our website. Kindly refer to the above comment for team details. Here's what we've done. 

Report:

1. Cleaning of data and csv preparation for all data-sets present and extracting graphs and uploading them for easier access to all. Secondary datasets have also been prepared where location co-ordinates and zipcodes have been added.

2. Geo-spatial mapping of tax collection in data for all given datasets. It facilitates easier understanding of trends and comparing taxes geospatially for multiple years. 

3. Uploading the above submissions on our website, making the website user-friendly and graphic for better operation. Designing layout of the website to easily access files. We plan to update this website with fresh data everytime its released. There is a separate blog started by the team on the website, although in its developmental stage, we plan to work on it more.

Link to our website: 

http://playcoders.wixsite.com/punepropertytax

Thank you and regards,
Team Playcoders.

**Comment by mufaddal12** [comment]
Cleaned csv version of dataset "Ward wise Property Tax collection for the year 2016--17".
Team 404NotFound
Members :
Mufaddal Diwan  
Aboli Marathe
Hussain Naharpura
https://drive.google.com/open?id=1fzcy5Ad0VtlvZ9DBo1TtrmWJMGilPFgd

**Comment by answerquest** [comment]
@harshadmanglani  team Playcoders, nice website!

**Comment by answerquest** [comment]
@mufaddal12 the link you shared is not a CSV, there's multiple header rows etc. A cleaned dataset would have only one header row, no merged cells etc. (explanation can be put in a readme doc to accompany the file)

**Comment by harshadmanglani** [comment]
@answerquest Sir, may I ask you for a little more feedback if we've gone wrong somewhere or lack something? I would kindly request you to go through everything on the website, such as the maps, graphs and other content. Thank you for your support and guidance.

Regards,
Team Playcoders.

**Comment by answerquest** [comment]
@harshadmanglani the lower sections with the stock photos (with foreign people) are a bit confusing.

**Comment by harshadmanglani** [comment]
@answerquest Sir, do you mean the photos of the graphs one after another? I didn't understand clearly.
